src/OrbitGgp/include/OrbitGgp/SymbolDownloadInfo.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GGP_SYMBOL_DOWNLOAD_INFO_H_
+#define ORBIT_GGP_SYMBOL_DOWNLOAD_INFO_H_
+
+#include <QByteArray>
+#include <QString>
+#include <QVector>
+
+#include "OrbitBase/Result.h"
+
+namespace orbit_ggp {
+
+struct SymbolDownloadInfo {
+  QString file_id;
+  QString url;
+
+  static ErrorMessageOr<QVector<SymbolDownloadInfo>> GetListFromJson(const QByteArray& json);
```

Comment:
I think we should use std::vector instead of QVector as I don't see any advantage of QVector in that case. Did I miss something?

Reply:
Changed to `std::vector`. Thanks for the suggestion